packages/celotool/src/lib/geth.ts

```diff
@@ -1147,3 +1147,64 @@ export async function connectValidatorPeers(instances: GethInstanceConfig[]) {
     instances.filter(({ wsport, rpcport, validating }) => validating && (wsport || rpcport))
   )
 }
+
+export async function migrateContracts(
+  monorepoRoot: string,
+  validatorPrivateKeys: string[],
+  attestationKeys: string[],
+  validators: string[],
+  to: number = 1000,
+  overrides: any = {},
+  verbose: boolean = true
+) {
+  const migrationOverrides = merge(
+    {
+      downtimeSlasher: {
+        slashableDowntime: 6,
```

Comment:
For better generality, this could depend on the epoch length - this number needs to be <= epoch length, otherwise migrations will fail.